src/libraries/System.IO.MemoryMappedFiles/tests/MemoryMappedFile.CreateNew.Tests.cs

```diff
@@ -112,7 +112,7 @@ public void TooLargeCapacity_Unix()
             // due to differences in OS behaviors and Unix not actually having a notion of
             // a view separate from a map.  It could also come from CreateNew, depending
             // on what backing store is being used.
-            Assert.Throws<IOException>(() =>
+            AssertExtensions.ThrowsAny<IOException, ArgumentException>(() =>
```

Comment:
It looks like this method is poorly named, in that whereas `Assert.ThrowsAny<T>` allows for the exception to be T or anything derived from it, `AssertExtensions.ThrowsAny<T1, T2>` is requiring the concrete type of T1 or T2:
https://github.com/dotnet/runtime/blob/22c3105575d6edf545c55e1773ff8b75b766f8b5/src/libraries/Common/tests/TestUtilities/System/AssertExtensions.cs#L195-L216
which is then failing if the exception is actually an ArgumentOutOfRangeException.